### ISSUE TITLE
file view not changing after user interaction

### ISSUE THREAD

**Issue by ivarsu** [scrimba/community#239]
in [this](https://scrimba.com/p/pWqLHa/cdp76sD) tutorial, my view of files don't change based on what the author is viewing, maybe this is done intentionally. But it would be sweet to have some kind of indicator as to what file is focused on by the author/teacher, or a restriction that always focuses on what the author/teacher is working on.

**Comment by judofyr** [comment]
Yes, this is currently done intentionally, but we're interested in feedback on how to improve the behaviour. We have a few options:

1. Changing file should automatically pause the tutorial. Resuming will bring you back to the author. Pro: You will always be following the author. Con: You can't quickly jump around without pausing.

2. `n` seconds after changing a file you will automatically jump back to the author's file. Pro: You can still jump around. Con: It jumps back out of your control.

3. There can be a separate button "Follow the author". Pro: You have full control if you want to follow the author or play around. Con: Another button which clutters the view; users can find the whole button confusing.

4. If you change back to the file the author is editing, you'll go back to following the author again. Combine this with a clear indicator of which file the author is editing.

Thoughts?

**Comment by ivarsu** [comment]
I like the sound of both n1 and n4, but in my opinion n1 is the most "beleilig". My reasoning is that the user shouldn't really be editing files while the theacer is teaching, also i think that this behaviour would be expected by most users. n4 is also interesting to satisfy the powerusers, or people who just need to brush up on the material. n3 and n2 would just (as you said) clutter up the experience. A choice between the two could be implemented inside of a settings page or/and the player spesific settings,
don't know if you have planned any settings page but i for one would love one, that is when there are settings to play with...
